File: presentation/01_Basics/basics.py
# Topic: Basic Control Flow
# Count 0-9 and identify Even/Odd

def main():
    i = 0
    while i < 10:
        if i % 2 == 0:
            print(f"Count: {i} (Even)")
        else:
            print(f"Count: {i} (Odd)")
        i += 1

if __name__ == "__main__":
    main()

File: presentation/01_Basics/basics.rs
// Topic: Basic Control Flow
// Count 0-9 and identify Even/Odd

fn main() {
    let mut i = 0;
    while i < 10 {
        if i % 2 == 0 {
            println!("Count: {} (Even)", i);
        } else {
            println!("Count: {} (Odd)", i);
        }
        i += 1;
    }
}
